### PR TITLE
People: New feature to mute posts and stories

### PR DESCRIPTION
**Pull request by mgp25**
This is working fine, just wanted to know if this is the best coding solution or if there is a better one.

Regards

### CODE REVIEW

**Comment by mgp25** [comment]
@valga I have followed all your recommendations, it looks nice! Yes, it can be used with current API key 😄

File: src/Request/People.php [hunk 1]
```diff
@@ -750,6 +750,89 @@ public function block(
             ->getResponse(new Response\FriendshipResponse());
     }
 
+    /**
+     * Mute stories, posts or both from a user. It presents user media from
```
Comment (valga) [inline src/Request/People.php]: Typo (preVents).

File: src/Request/People.php [hunk 2]
```diff
@@ -750,6 +750,89 @@ public function block(
...
+        return $this->_muteOrUnmuteUserMedia($userId, $option, 'friendships/unmute_posts_or_story_from_follow/');
+    }
+
+    /**
+     * Helper function to mute user media.
+     *
+     * @param string $userId   Numerical UserPK ID.
+     * @param string $option   Selection of what type of media are going to be muted.
+     *                         Available options: 'story', 'post' or 'all'.
+     * @param bool   $endpoint API endpoint for muting/unmuting user media.
```
Comment (valga) [inline src/Request/People.php]: Argument type has been changed to `string`.

**Comment by mgp25** [comment]
@valga Im going to test that scenario in an hour or so and confirm the result.

**Comment by mgp25** [comment]
@valga Yes, you still receive a notification.

**Pull request merged**